### ISSUE TITLE
Migration of image hosting from gcr.io to Artifact Registry

### ISSUE THREAD

**Issue by akdigitalself** [knative/infra#85]

**Comment by kvmware** [comment]
@upodroid are we done with this issue now?

**Comment by upodroid** [comment]
No, we completed the migration for the internal CI. Our public images for end users need to be planned and started soon.

**Comment by thesuperzapper** [comment]
@upodroid did this migration ever end up happening, because it's not clear that the images will still exist after 2025 otherwise?
